### PR TITLE
Correct description of Pro integration branch architecture

### LINKED ISSUE #9062: Fix Cloud Pro integration branch description
# General issue

## Description:

A number of PaaS containers for Cloud Pro is confusing.

https://devdocs.magento.com/cloud/architecture/pro-architecture.html:

> Integration—Provides a single environment branch, and you can create one additional, environment branch. 
This allows for up to **two active** branches deployed to Platform-as-a-Service (PaaS) containers.

https://devdocs.magento.com/cloud/architecture/pro-develop-deploy-workflow.html#develop:

> You can create up to four additional branches for developing your custom code, extensions, and third party integrations. This allows for a maximum of **five active** branches deployed to Platform-as-a-Service (PaaS) containers.

https://magento-u.magento.com/magento-u/Free-Intro-to-Magento-Commerce-Cloud/web-free-intro-to-magento-commerce-cloud-raw-fv6qFSHv/content/index.html#/lessons/zLyv-85EA5n19YQK3bcVMxFmeulnUf7h:

image

## Possible solutions:

Update https://devdocs.magento.com/cloud/architecture/pro-architecture.html with **5** branches instead of **2**.

Thank you for taking the time to report this issue!
GitHub Issues should only be created for problems/topics related to this project's codebase.

Before submitting this issue, please make sure you are complying with our Code of Conduct:
https://github.com/magento/devdocs/blob/master/.github/CODE_OF_CONDUCT.md

Issues that do not comply with our Code of Conduct or do not contain enough information may be closed at the maintainers' discretion.

Feel free to remove this section before creating this issue.
-->

### PR DESCRIPTION
**Pull request by meker12**
Fixes #9062 

## Purpose of this pull request

This pull request corrects information about the number of integration branches for Cloud Pro environments.

## Affected DevDocs pages

-  [Pro develop and deploy workflow](https://devdocs.magento.com/cloud/architecture/pro-develop-deploy-workflow.html#develop)

### CODE REVIEW

File: src/cloud/architecture/pro-develop-deploy-workflow.md [hunk 1]
```diff
@@ -17,7 +17,9 @@ The following graphic demonstrates the Pro develop and deploy workflow, which us
 
 ## Development workflow {#develop}
 
-The Integration environment provides a single, base `integration` branch containing your {{site.data.var.ece}} code. You can create up to four additional branches for developing your custom code, extensions, and third party integrations. This allows for a maximum of five _active_ branches deployed to Platform-as-a-Service (PaaS) containers.
+The Integration environment provides a single, base `integration` branch containing your {{site.data.var.ece}} code. You can create one additional, environment branch. This allows for up to two active branches deployed to Platform-as-a-Service (PaaS) containers.(PaaS) containers.
```
Comment (BarnyShergold) [inline src/cloud/architecture/pro-develop-deploy-workflow.md:20]:
```suggestion
The Integration environment provides a single, base `integration` branch containing your {{site.data.var.ece}} code. You can create one additional environment branch. This allows for up to two active branches deployed to Platform-as-a-Service (PaaS) containers.(PaaS) containers.
```

Comment (BarnyShergold) [reply src/cloud/architecture/pro-develop-deploy-workflow.md:20]: Should this not mention that its one additional active environment? You can have as many inactive environments as you want. Or is this just complicating the docs?

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

**Comment by hguthrie** [comment]
@meker12 Just for future reference, can you provide a comment about the reason for closing?

**Comment by meker12** [comment]
Clean up branches in public repo. Small updates in this PR incorporated in internal PR.

**Pull request closed without merging**